react-class-components/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-class-components",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.20.5",
+    "@babel/plugin-transform-react-jsx": "^7.19.0",
+    "babel-loader": "^9.1.0",
```

Comment:
you also need to install webpack and webpack-cli

Also, the only `dependencies` should be react and react-dom

All other dependancies should be installed as `dev dependencies`